.github/workflows/danger.yml

```diff
@@ -9,62 +9,67 @@ jobs:
     name: Run Danger
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 100
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.5.0
```

Comment:
I wish Github Actions was doing that by default 😢 Bitbucket Pipeline also does not cancel builds.

Reply:
I think they already had the plan to do it.
This action is recommended by Github staff
https://github.community/t/github-actions-cancel-redundant-builds-not-solved/16025/31